### PR TITLE
hal push: don't push stale data

### PR DESCRIPTION
**Pull request by MJedr**
ref: #2008

### CODE REVIEW

File: backend/inspirehep/hal/api.py [hunk 1]
```diff
@@ -25,7 +25,11 @@ def push_to_hal(record):
 
     if is_hal_set(record):
         current_celery_app.send_task(
-            "inspirehep.hal.tasks.hal_push", kwargs={"recid": record["control_number"]}
+            "inspirehep.hal.tasks.hal_push",
+            kwargs={
+                "recid": record["control_number"],
+                "record_version_id": record.model.version_id,
+            },
```
Comment (michamos) [inline backend/inspirehep/hal/api.py:32]: the TECH NOTES in the issue talk about moving the trigger from the record class to the receivers. Why did you change the approach?

Comment (MJedr) [reply backend/inspirehep/hal/api.py:32]: It was discussed with @drjova, the issue is that we need to pass the feature flag which is disabling push in some cases (for update).

Comment (michamos) [reply backend/inspirehep/hal/api.py:32]: that flag was used only in the migrator, and we don't use that any longer.

Comment (MJedr) [reply backend/inspirehep/hal/api.py:32]: That's not true, it is also used in hal push:
https://github.com/inspirehep/inspirehep/blob/4e47de34dc2e0500772425e9fa6687722dc127ac/backend/inspirehep/hal/tasks.py#L110

Comment (michamos) [reply backend/inspirehep/hal/api.py:32]: ok, my bad, I hadn't realized that :+1:

**Review by drjova — APPROVED** [review]
